workloads/templates/workload-nodevertical-script-cm.yml.j2

```diff
@@ -98,6 +116,14 @@ data:
     end_time=$(date +%s)
     duration=$((end_time-start_time))
 
+    if [ "${PPROF_COLLECT}" = "true" ]; then
+      workload_log "Stopping conprof"
+      bash /tmp/conprof_stop.sh
+      cp /tmp/conprof.log ${result_dir}/conprof.log
+      cp /tmp/conprof.yaml ${result_dir}/conprof.yaml
+      tar -czvf ${result_dir}/conprof.tar.gz /tmp/data/
```

Comment:
Maybe add a step in the playbook to run `$ oc cp ` to copy the data from /tmp of the pod in case pbench_instrumentation is not set? This way it works with both pbench enabled/disabled. Thoughts?

Reply:
@chaitanyaenr that's a nice idea, do we currently do this with any other workload to copy data? if so, can you please point me to it, so that i can try to adapt it here. I've a few concerns about trying to do this obviously, especially around how do we orchestrate job completion from outside the pod, but if there's a way we're doing this already then i can just try to do the same.

Reply:
Resolving the conversation as per the discussion to implement a bottle based server for handling the data to work even when pbench is disabled. Thanks.